tests/catalog/test_memory.py

```diff
@@ -607,7 +400,7 @@ def test_list_tables(catalog: InMemoryCatalog) -> None:
     # Given
     given_catalog_has_a_table(catalog)
     # When
-    tables = catalog.list_tables()
+    tables = catalog.list_tables(namespace=TEST_TABLE_NAMESPACE)
```

Comment:
nit: i think for this test we want to call `catalog.list_tables()` (without the namespace parameter) to return all tables

Reply:
nevermind, as part of #1163, `list_table` should always take a parameter